src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img-wrapper">
+        <img
+          src="images/imac.jpeg"
+          class="card__img"
+          alt="product picture"
+        >
+      </div>
+      <div class="card__product-details">
+        <p class="card__product-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="card__code">
+          Код товара: 195434
+        </p>
+      </div>
+      <div class="card__comment-section">
+        <div class="card__star-row">
+          <div class="card__star-single"></div>
+          <div class="card__star-single"></div>
+          <div class="card__star-single"></div>
+          <div class="card__star-single"></div>
+          <div class="card__star-single"></div>
+        </div>
+        <div class="card__comments-number">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="card__price-section">
+        <div class="card__price-word">
+          Цена:
+        </div>
+        <div class="card__price-value">
+          69 999 грн
+        </div>
+      </div>
+      <div class="card__button card__button--hover">
```

Comment:
```suggestion
      <div class="card__button">
```
You don't need another class for hover